src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -102,7 +102,7 @@ function WorkspaceMoreFeaturesPage({policy, route}: WorkspaceMoreFeaturesPagePro
             titleTranslationKey: 'workspace.moreFeatures.taxes.title',
             subtitleTranslationKey: 'workspace.moreFeatures.taxes.subtitle',
             isActive: (policy?.tax?.trackingEnabled ?? false) || isSyncTaxEnabled,
-            disabled: isSyncTaxEnabled || policy?.connections?.quickbooksOnline.data.country === CONST.COUNTRY.US,
+            disabled: isSyncTaxEnabled || policy?.connections?.quickbooksOnline?.data.country === CONST.COUNTRY.US,
```

Comment:
Seems like this should fix [the crash](https://expensify.slack.com/archives/C01GTK53T8Q/p1714148778811739?thread_ts=1714119216.357459&cid=C01GTK53T8Q)
```suggestion
            disabled: isSyncTaxEnabled || policy?.connections?.quickbooksOnline?.data?.country === CONST.COUNTRY.US,
```

Reply:
I've implemented it and it should in fact fix the crash issue after testing.